packages/client/components/MyDashboardTasks.tsx

```diff
@@ -29,7 +30,7 @@ const MyDashboardTasks = (props: Props) => {
       <UserTasksHeader viewer={viewer} />
 
       {showArchivedTasksCheckbox ? (
-        <PersonalTaskArchiveRoot teamId={teamFilter?.id} team={teamFilter} />
+        <ArchiveTaskRoot teamIds={teamFilter ? [teamFilter.id] : null} userIds={[viewerId]} team={teamFilter} />
```

Comment:
the nuance here is that `teamIds` and `userIds` are filters. if one is not set, that's like saying "don't filter this thing".

So, reading this component we can say "filter to just the viewer's tasks, and don't filter out any teams"